<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dtuonefixmodifyClass</class>
 <widget class="QWidget" name="dtuonefixmodifyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>275</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>275</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>点表修改</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="text_1">
       <property name="text">
        <string>当前点表号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_2">
       <property name="text">
        <string>十六进制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_hex">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_3">
       <property name="text">
        <string>十进制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_dec">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="text_4">
     <property name="text">
      <string>当前点表描述:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLabel" name="lab_desc">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="text_5">
     <property name="text">
      <string>点表附加描述:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLabel" name="lab_adddesc">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="text_6">
       <property name="text">
        <string>输入新点表值:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_8">
       <property name="text">
        <string>十六进制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_new_hex">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_9">
       <property name="text">
        <string>十进制</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_new_dec">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lab_state">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btn_check">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>验证</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btn_modify">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>修改</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QLabel" name="text_7">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>注意:只可输入十进制或十六进制,十六进制以0x开始</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_check</sender>
   <signal>clicked()</signal>
   <receiver>dtuonefixmodifyClass</receiver>
   <slot>checkid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_modify</sender>
   <signal>clicked()</signal>
   <receiver>dtuonefixmodifyClass</receiver>
   <slot>modifyfixid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>modifyfixid()</slot>
  <slot>checkid()</slot>
 </slots>
</ui>
